src/IronBeard.Core/Features/Markdown/MarkdownProcessor.cs

```diff
@@ -56,7 +68,7 @@ public async Task<OutputFile> ProcessAsync(InputFile file)
             var result = this.ExtractYamlMetadata(markdown);
 
             // convert markdown to HTML
-            var html = Markdig.Markdown.ToHtml(result.markdown);
+            var html = Markdig.Markdown.ToHtml(result.markdown, _pipeline);
```

Comment:
@road42 Just want to double check, if the user hasn't configured the config to set `EnableMarkdownExtensions = true`, the `_pipeline` variable here would be null. `Markdig.Markdown.ToHtml(result.markdown, _pipeline)` supports passing in a null value for the second parameter?

Reply:
I answered my own question be checking out their open source project. Seems to handle it correctly:
```csharp
public static string ToHtml(string markdown, MarkdownPipeline pipeline = null)
        {
            if (markdown == null) throw new ArgumentNullException(nameof(markdown));
            var writer = new StringWriter();
            ToHtml(markdown, writer, pipeline);
            return writer.ToString();
        }
```

Reply:
Yes, default value for the pipeline is null. I tested this.

Also if you don‘t add the configuration parameter to the beard.json everything works.